cip-drep-metadata/README.md

```diff
@@ -0,0 +1,248 @@
+- It is expected that the "Identity" of a DRep will be the addresses of their most active social media account twitter, linkedin etc. or personal website.
+- The DRep must reference their DRep ID in a prominent place at the location that is specified in the `uri` property of that reference. This will be used by people reviewing this DRep to prove and verify that the person described in the metadata is the same as the person who set up the social media profile.
+- Tooling providers making metadata SHOULD provide some information about why this is important.
+
+PKI cryptographic solutions were also considered in the place of the current solution however they were generally considered too complex to implement for minimum viable tooling. There were also other issues with cryptographic forms of identification for MVPs:
+1. solutions that involve a public/private key setup still require people verifying the identity of a DRep to know what their authentic public key is.
+2. specifying the use of a verification service such as [Keybase](https://keybase.io/) would lead to centralisation and therefore reliance on a 3rd party for the identity validation of DReps.
+
+#### `donotlist`
+- Optional
+- If not included then the value is assumed to be false
+- A boolean expression
+- A true value means that the DRep does not want to show up in tooling that displays DReps. 
+-- I.e. they do not want to appear in GovTool’s DRep Explorer feature
+
+### Application
+DRep metadata must include all compulsory fields to be considered CIP-???? compliant. As this is an extension to CIP-100, all CIP-100 fields can be included within CIP-???? compliant metadata.
+
+### Test Vector
+See test-vector.md for examples.
+
+### Versioning
+This proposal should not be versioned, to update this standard a new CIP should be proposed. Although through the JSON-LD mechanism further CIPs can add to the common governance metadata vocabulary,
+
+## Rationale: how does this CIP achieve its goals?
+We intentionally have kept this proposal brief and uncomplicated. This was to reduce the time to develop and deploy this standard. This way we enable tooling which depends on this standard to start development. The fields which have been chosen for this standard are heavily inspired by those that we are seeking to introduce for GovTool. We did this because GovTool will likely be the first technical implementation of this standard. 
+
+I believed that this CIP would provide a benefit to:
+
+#### Potential delegators
+When observing from the chain level, tooling can only see the content and history of DRep registration and update certificates and any associated anchors. These on-chain components do not give any context to the motivation of a DRep, even though this information would likely be the desired context for people who might delegate their voting power. By providing rich contextual metadata we enable people choosing a DRep to delegate their voting power to make well informed decisions.
```

Comment:
This is more `Motivation` than `Rationale`

Reply:
dup https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1549605293